integration-cli/docker_cli_rmi_test.go

```diff
@@ -14,26 +15,22 @@ func (s *DockerSuite) TestRmiWithContainerFails(c *check.C) {
 
 	// create a container
 	out, _, err := dockerCmdWithError("run", "-d", "busybox", "true")
-	if err != nil {
-		c.Fatalf("failed to create a container: %s, %v", out, err)
-	}
+	// failed to create a container
+	c.Assert(err, checker.IsNil, check.Commentf("out: %s", out))
```

Comment:
@WeiZhang555 yes it is ! You don't ignore failure of `docker run`, they are "handled" in `dockerCmd` (see https://github.com/docker/docker/blob/master/pkg/integration/dockerCmd_utils.go#L33, there is an Assert on the err :wink:)